__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const { app } = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const connection = require("../db/connection");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  connection.end();
+});
+
+describe("API Testing", () => {
+  describe("Categories", () => {
```

Comment:
Is your intention here of having a seperate describe block for each endpoint with a nested describe for the different request types? 

If so I like that this is all being organised together but the endpoint path ' /api/categories' may be a little clearer for the description than 'categories'

Reply:
this makes sense, will do that!